about/coronavirus_return_policy.md

```diff
@@ -0,0 +1,83 @@
+
+There are 2 main routes by which people can spread COVID-19:
+
+- infection can be spread to people who are nearby (within 2 metres) or possibly could be inhaled into the lungs
+- it is also possible that someone may become infected by touching a surface, object or the hand of an infected person that has been contaminated with respiratory secretions and then touching their own mouth, nose, or eyes (such as touching contaminated surfaces (e.g. door knob) or shaking hands then touching own face)
+
+## Organisational Named Person For COVID-19 Response
+
+Best practices dictate that organisations should have a named person responsible for their response to the COVID-19 pandemic.  
+
+As such, Andrew Bolster will be responsible for these guidelines, will be the contact person for questions regarding our response to COVID-19 and will be responsible for organisational contact tracing in the event of a member reporting an infection. ([bolster@farsetlabs.org.uk](mailto:bolster@farsetlabs.org.uk) / 07783249547).
+
+Should Andrew be unable to fulfil this role for any reason Colin Mitchell, will deputise ([colin.mitchell@farsetlabs.org.uk](mailto:colin.mitchell@farsetlabs.org.uk)) 
+
+## Guidelines for Reopening of Farset Labs
+
+Farset Labs will begin a phased reopening on 22/06/20 at 10am. The reopening will proceed based on the following guidance:
+
+- Upon reopening we will request that our members continue to work from home if they can, in line with the [guidance from the NI Executive](https://www.executiveoffice-ni.gov.uk/sites/default/files/publications/execoffice/execuitveour-approach-to-decision-making.pdf). 
+- Access to the building will be subject to these new guidelines.
+- It will be left to the discretion and good sense of individual members to decide whether they can continue to work from home or need to use the space.
+- All members will be required to complete and sign an online Pre-Return form which indicates their agreement with these guidelines. This must be done before they can return to the space.
+
+If you wish to return to the space at this time, you must contact Andrew Bolster by email ([bolster@farsetlabs.org.uk](mailto:bolster@farsetlabs.org.uk)) to notify him of your intention to return. You will then be asked to fill in a Pre-Return Disclaimer, after which you will be given new access control codes.
+
+**Do not return to the space if you have not already digitally signed the pre-return disclaimer as you will not be permitted in the building**
+
+## Guidelines For Access To Farset Labs During Ongoing Covid-19 Restrictions
+
+It is anticipated that social distancing restrictions will remain in place for some time. In order to respect these restrictions and to ensure the levels of cleanliness required to prevent infection members will be expected to adhere to the following guidelines.
```

Comment:
```suggestion
It is anticipated that social distancing restrictions will remain in place for some time. In order to respect these restrictions and to ensure the levels of cleanliness required to prevent infection members will be expected to adhere to the following guidelines;
```